deploy.sh

```diff
@@ -24,26 +26,13 @@ case "${1:-}" in
 esac
 
 
-if [[ -z "${RESOURCE}" || -z "${AWS_ACCESS_KEY_ID}" || -z "${AWS_SECRET_ACCESS_KEY}" ]]; then
-  echo "environment variables are empty, exiting..."
-  exit 1
-fi
-
-
 echo "build and push frontend"
 (cd frontend && yarn run build && cp -r dist ../public)
 aws s3 sync --delete public "s3://codeforpoznan-public/${RESOURCE}"
 
 
 echo "refresh CDN"
-dist_id=$(
-  aws cloudfront list-distributions \
-    | jq --arg PATH "/${RESOURCE}" -r '
-      .DistributionList.Items[]
-      | {Id, p: .Origins.Items[].OriginPath}
-      | select(.p == $PATH).Id'
-)
```

Comment:
I'm removing (or rather, reverting) this bit because IAM user we're using does not have the permission to list cloudfront distros, and we don't want to give that permission to it, because it's not worth it - instead we'll use another env var and type the distro id directly in there.